src/main/java/core/basesyntax/interfaces/AreaCalculator.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax.interfaces;
+
+public interface AreaCalculator {
+    double area();
```

Comment:
```suggestion
    double getArea();
```

Reply:
Tell me please. You said above. I need to delete interface public class Circle extends Figure implements AreaCalculator.
I need to rename the method and move it to abstract class?

Reply:
you've done right